### PR TITLE
fix migration guide v4 for yarn workspaces

### PR DESCRIPTION
**Pull request by zemlanin**
`yarn` workspaces can place package either in `{REPO_ROOT}/node_modules` or in the package directory (e.g. `{REPO_ROOT}/packages/{PACKAGE_NAME}/node_modules`)

Because it depends on version resolutions, dependency upgrade in one package could break another's relative paths to the `node_modules`. `import "yoshi/types"` solves this problem by offloading path resolution to `node`

---

related [PR](https://github.com/wix-private/site-search/pull/217) and [commit](https://github.com/wix-private/site-search/pull/217/commits/a3aa78e534e3e969a5ef85e16363be0c1c5f51d5)

### CODE REVIEW

File: docs/migration/version-4.md [hunk 1]
```diff
@@ -87,6 +83,8 @@ Finally, remove any old types you had on assets handled by Yoshi (normally in `s
 ```diff
 -declare module '*.scss';
 -declare module '*.json';
++import "wix-ui-core/dist/src/globals";
```
Comment (bodia-uz) [inline docs/migration/version-4.md]: looks like `import "wix-ui-core/dist/src/globals"` should not be here.

File: docs/migration/version-4.md [hunk 2]
```diff
@@ -75,10 +75,6 @@ Then, change your `tsconfig.json`:
 +    "resolveJsonModule": true,
     "..."
   },
-  "files": [
-+    "./node_modules/yoshi/types.d.ts",
-    "..."
-  ]
 }
 ```
```
Comment (bodia-uz) [inline docs/migration/version-4.md]: Need to update text about "remove any old types" to smth like "replace old types with types from yoshi"

File: docs/migration/version-4.md (hunk 1 above)
Comment (zemlanin) [reply docs/migration/version-4.md]: Yeah, confused typings for `*.st.css` with the ones for `*.scss`

File: docs/migration/version-4.md (hunk 2 above)
Comment (zemlanin) [reply docs/migration/version-4.md]: https://github.com/wix/yoshi/pull/1492/commits/82dd4fbf812560a848307e541d8b7015ced9d83e

File: docs/migration/version-4.md (hunk 1 above)
Comment (zemlanin) [reply docs/migration/version-4.md]: https://github.com/wix/yoshi/pull/1492/commits/82dd4fbf812560a848307e541d8b7015ced9d83e

File: docs/migration/version-4.md [hunk 3]
```diff
@@ -75,18 +75,15 @@ Then, change your `tsconfig.json`:
...
 
-Finally, remove any old types you had on assets handled by Yoshi (normally in `src/external-types.d.ts`):
+Finally, replace old types you had on assets (normally in `src/external-types.d.ts`) to [the ones provided by Yoshi](https://github.com/wix/yoshi/blob/v4.11.1/packages/yoshi/types.d.ts):
```
Comment (bodia-uz) [inline docs/migration/version-4.md]: Not sure, but mb relative link will be better. Pros - latest source. Cons is that if it moved to another path, link will be broken.
```suggestion
Finally, replace old types you had on assets (normally in `src/external-types.d.ts`) to [the ones provided by Yoshi](/packages/yoshi/types.d.ts):
```

Comment (bodia-uz) [reply docs/migration/version-4.md]: Ah. If it used outside github(yoshi site), direct link is better.

Comment (zemlanin) [reply docs/migration/version-4.md]: Con outweighs pro, imo

**Pull request merged**